mock-identity-system/src/main/java/io/mosip/esignet/mock/identitysystem/dto/Password.java

```diff
@@ -0,0 +1,24 @@
+package io.mosip.esignet.mock.identitysystem.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
```

Comment:
please remove this dto, as it is no longer needed.

Reply:
removed